test/functional/page_objects/time_picker.ts

```diff
@@ -92,7 +92,9 @@ export class TimePickerPageObject extends FtrService {
    * @param option 'Today' | 'This_week' | 'Last_15 minutes' | 'Last_24 hours' ...
    */
   async setCommonlyUsedTime(option: CommonlyUsed | string) {
+    await this.testSubjects.exists('superDatePickerToggleQuickMenuButton', { timeout: 5000 });
     await this.testSubjects.click('superDatePickerToggleQuickMenuButton');
+    await this.testSubjects.exists('`superDatePickerCommonlyUsed_${option}`', { timeout: 5000 });
```

Comment:
Should this only use the backticks?
```suggestion
    await this.testSubjects.exists(`superDatePickerCommonlyUsed_${option}`, { timeout: 5000 });
```

Reply:
Wait! How on earth did that not fail? will review this